styleguide/source/assets/scss/02-molecules/_article-stub.scss

```diff
@@ -0,0 +1,67 @@
+.ama__article-stub {
+
+  &--container-image {
+    height: auto;
+    width: 100%;
+  }
+
+  &__copy {
+    flex-grow: 1;
+  }
+
+  &--related {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    align-items: center;
+    @include ama-rules(1px, '', solid, $gray-50);
+    @include gutter($padding-top-half...);
+    @include gutter($padding-bottom-half...);
+
+    .ama__image,
+    .ama__video {
+      @extend .col-xs-3;
+      & > div {
+        display: flex;
+        align-items: center;
+      }
+    }
+
+    .ama__article-stub__copy {
+      display: flex;
+      flex-direction: column;
+      @extend .col-xs-9;
+      @include gutter($padding-left-half...);
+      padding-top: 0;
+    }
+  }
+
+  &--small {
```

Comment:
This is not floating the image appropriately. This is probably a good use case for floats as opposed to flexbox.

<img width="795" alt="screen shot 2018-01-24 at 4 17 31 pm" src="https://user-images.githubusercontent.com/1653723/35360123-45bcbeaa-0122-11e8-9964-519d6f4a418a.png">

Reply:
the content just needs a width with flex-grow

Reply:
fixed